Core/Utils/AutoAttack.cs

```diff
@@ -131,6 +132,12 @@ public static float GetRealAutoAttackRange(this AttackableUnit target)
             var result = GameObjects.Player.AttackRange + GameObjects.Player.BoundingRadius;
             if (target != null && target.IsValid)
             {
+                var targetBase = target as Obj_AI_Base;
+                if (targetBase != null && GameObjects.Player.ChampionName == "Caitlyn"
+                    && targetBase.HasBuff("caitlynyordletrapinternal"))
```

Comment:
I think it does bcz i steal it from Sebby and Xcsoft, both of them only use that buff name